### ISSUE TITLE
Sponsor thanks should be smaller (font) and not at the bottom

### ISSUE THREAD

**Issue by Aterfax** [SLUGSoc/website-old#44]
User feedback:

> Ending the page with the join now section is perfect.

> I'd put the sponsors thanks as a smaller section.

> How does the font thanking them rate being larger than text describing the society, introducing socials, or introducing lans?
> Size to me denotes importance and sponsors just aren't as important as critical society information

**Comment by boardfish** [comment]
> Ending the page with the join now section is perfect.

I'm buying this. Creating a separate issue to repeat the join link at the bottom.

> I'd put the sponsors thanks as a smaller section.

Not sure, but I might halve the column and have the other half taken up as above.

>Size to me denotes importance and sponsors just aren't as important as critical society information

I'd like to think so too, but don't we at least owe them something? It'd look weird to have this section any smaller.

**Comment by Aterfax** [comment]
You can simply give the sponsors their own page?

**Comment by boardfish** [comment]
If we shunt them off over there, nobody's actively going to look at it, and it doesn't hold up our side of the bargain. It's like putting your ads on their own page.

**Comment by Aterfax** [comment]
Shifting their logo completely away is a no no - I just meant in terms of further text or detail - they should have their own page.

It should still have their icons clearly displayed on the front page / all pages if feasible.

**Comment by boardfish** [comment]
That's a reasonable shout, but we don't have enough sponsors right now to warrant an entirely new page for them. I'm not sure what the best course of action is beyond leaving them at the foot of the index page. Closing this until justifiable.

**Comment by guruthree** [comment]
For now as there's only the one sponsor(?) I'd suggest splitting it into two columns. Move "A Massive Thanks[..,]" to the left then put "Molly Malones" on the right. Then maybe just make all the fonts involved one size smaller.

**Comment by boardfish** [comment]
Like this? 
![2018-09-09-130210_1920x1080_scrot](https://user-images.githubusercontent.com/20680337/45264202-d3d23a00-b430-11e8-8b4b-c7349ba61601.png)

**Comment by guruthree** [comment]
Yeah. It'd be good if the grey background could extend to cover the left to make it clear those two things are related.
